packages/amplify-util-mock/src/api/api.ts

```diff
@@ -78,6 +83,21 @@ export class APITest {
     this.resolverOverrideManager.stop();
   }
 
+  async checkJavaversion() {
+    var javaSpawn = spawnSync('java', ['-version'], {
```

Comment:
Can you make sure spawning 'java' on windows does not throw an error

Reply:
yes , I tested it, it works